tests/data_store_tests/validation_tests/pathfinders/schema_validation_tests/test_pf_schema_validate.py

```diff
@@ -101,8 +101,11 @@ def dataframe_with_missing_indexes():
 
 
 def build_mock_extracted_table(data: dict[str, list[Any]]) -> Table:
-    data = pd.DataFrame(data=data, index=range(len(list(data.values())[0])))
```

Comment:
Function has `data` as `dict[str, list[Any]]`. Reassigning it with a different type upsets mypy. I think reassigning in general upsets mypy. Don't upset mypy.